src/matrix/inversewishart.jl

```diff
@@ -1,8 +1,8 @@
 """
     InverseWishart(ν, Ψ)
 ```julia
-ν::Real   degrees of freedom (greater than p - 1)
-Ψ::PDMat  p x p scale matrix
+ν::Real           degrees of freedom (greater than p - 1)
```

Comment:
hit tab by error?

Reply:
not by error, but I can eliminate if you want.

Reply:
is it to align with the following lines?

Reply:
yeah